### DISCUSSION TITLE
Images added onto the storefront view is not showing

### DISCUSSION THREAD

**Discussion by jackomcpie** [solidusio/solidus#5062] [Troubleshooting]
Hello, I am fairly new to ruby on rails 7 and currently experimenting with solidus and spree. However, when i tried to add images to be shown on for example the home page or about page. It does not render, but it just shows a "broken" image icon. Ubuntu terminal also showed a ActionController::RoutingError (No route matches [GET] "/about/about.png"):. I placed my images in the app/assets/images directory and I used the img src="about/about.png" alt='about' tag as the <%= image_tag %> doesnt work for me, it only renders as a string. Anyone knows how to fix this? Thank you. 

![Screenshot 2023-05-03 080738](https://user-images.githubusercontent.com/132438276/235862772-dcd10831-984d-4a3e-ba50-51dddf292cb4.png)

**Comment by jarednorman** [comment]
This board is for troubleshooting issues with Solidus. It sounds like you need help with the Rails asset pipeline. You may be better off looking for help somewhere Rails-specific. For example, Rails' official forum recommends people seek help [on Stack Overflow](https://stackoverflow.com/questions/tagged/ruby-on-rails).

As you're not troubleshooting anything with Solidus, I'm going to close this, but here's a suggestion to help point you in the right direction:

It would be best if you used the `image_tag` helper. For example, if the file is placed at `app/assets/images/about/about.png`, you should render the image with `<%= image_tag "/about/about.png" %>`.
